### PR TITLE
refactor(flow): add skip auth to flow mapper

### PR DESCRIPTION
**Pull request by ungreat**
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yizhouxw — COMMENTED** [review]
seem too complex to add so much skipAuth parameter,
can we just change authorize logic instead ?

**Comment by ungreat** [comment]
> seem too complex to add so much skipAuth parameter, can we just change authorize logic instead ?

Currently, the method to bypass authentication is by passing parameters, which is indeed too complicated. In the future, it could be simplified by adopting a technical change to uniformly switch to using ThreadLocal.



> seem too complex to add so much skipAuth parameter, can we just change authorize logic instead ?

It‘s  true that most method to skip auth is by passing parameters at present, may need a technical refactor issue to deal with it uniformly.

**Review by yizhouxw — APPROVED** [review]

**Review by yizhouxw — APPROVED** [review]

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/FlowInstanceDetailResp.java [hunk 1]
```diff
@@ -96,6 +97,7 @@ public class FlowInstanceDetailResp {
     private Date completeTime;
     private List<FlowNodeInstanceDetailResp> nodeList;
 
+    @Getter
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/FlowInstanceDetailResp.java:100]: this getter seems useless?

**Review by yhilmare — APPROVED** [review]
lgtm

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/FlowInstanceDetailResp.java (hunk 1 above)
Comment (yhilmare) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/FlowInstanceDetailResp.java:100]: you may remove it in another pr~

**Pull request merged**